front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -178,6 +189,23 @@ const RenderNode = ({ render }) => {
         </Box>
       )}
       <div style={{ pointerEvents: name === IFRAME ? 'none' : 'auto' }}>
+        {name === IMAGE && !props.imageUrl && (
+          <Box
```

Comment:
I don't think this box is needed. You can play with the paddings in the image directly and set `margin: auto` so center it?

Reply:
I tried removing it, but I think we might need it to set the placeholder div to 100% width? Since the icon needs it's width set to a specific value? Or maybe I missed something?

Reply:
Couldn't you set the top div to width 100%?

Reply:
Ah of course! I was thinking the next level up was the component. Yes that should work - thanks :)